doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -21,24 +21,24 @@ mongooseim   hard   nofile   1000000
 
 If you are using **Ubuntu**, all `/etc/pam.d/common-session*` files should include `session required pam_limits.so`.
```

Comment:
Why?

Reply:
Why not? This line I didn't touch and from what I remember this worked when the page was first created.

Reply:
Typically there are `common-session` and `common-session-noninteractive`. Both should have the mentioned line if we want to be sure that the limits will be changed. If someone knows only one will be best for it then I assume this is quite experienced user who knows his system and will set it as he needs.
As you can see above the limits are only for user `mongooseim` so other users will connect with their limits anyway.

Reply:
Sorry, for a moment I was sure it was related to PAM auth backend that we've removed long ago. :)